lib/mpl_toolkits/axisartist/tests/test_grid_helper_curvelinear.py

```diff
@@ -75,7 +75,7 @@ def inverted(self):
     ax1.grid(True)
 
 
-@image_comparison(['polar_box.png'], style='default', tol=0.04)
+@image_comparison(['polar_box.png'], style='default', tol=0.27)
 def test_polar_box():
     # Remove this line when this test image is regenerated.
```

Comment:
Possibly add a note about re-evaluating `tol` when the test image is regenerated? (Same for the other similar change.)

Reply:
Sure, done.